include/properties/property_list.h

```diff
@@ -18,6 +18,8 @@ class property_list {
     std::unordered_map<std::string, std::string> unknown_props;
 
 public:
+    char sep = '=';
+
     property_list() {}
```

Comment:
I prefer to make sep a private field
```suggestion
    char sep = '=';

public:
    property_list() {}
    
    property_list(char sep) : sep(sep) {}
```
```